<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <widget class="PhVideoView" name="_videoView" native="true"/>
    </item>
    <item>
     <widget class="PhMediaPanel" name="mediaController" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionNext_frame"/>
    <addaction name="actionPrevious_frame"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="text">
    <string>Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionNext_frame">
   <property name="text">
    <string>Next frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious_frame">
   <property name="text">
    <string>Previous frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhVideoView</class>
   <extends>QWidget</extends>
   <header>PhVideo/PhVideoView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhMediaPanel</class>
   <extends>QWidget</extends>
   <header>PhCommonUI/PhMediaPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
